<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SalonAppointment</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Enter Your Full Name</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="nameEdit"/>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Select Date &amp; Time</string>
           </property>
          </widget>
          <widget class="QDateTimeEdit" name="dateTimeEdit">
           <property name="minimumDateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>1752</year>
             <month>9</month>
             <day>14</day>
            </datetime>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Select Stylist</string>
         </property>
        </widget>
        <widget class="QListWidget" name="stylistList"/>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Select Services</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="servicesList"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="makeAppBtn">
       <property name="text">
        <string>Make Appointment</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
